### ISSUE TITLE
Add Dialog UI Component

### ISSUE THREAD

**Issue by hollowaykeanho** [ZORALab/Hestia#22]
# Description
```
Please provide a short description of what feature you're looking forward to
add into the existing repository below. Please include the story behind your
idea as well to give a better visualization of your idea.
```
Dialog (also known as modal in web UI design) is a crucial prompts for a lot of applications. We have to develop one for it.
Also, all browser (exclude IE as it is marked retired by Microsoft) have already supported the feature: https://caniuse.com/dialog.

# Expected Behavior
```
Please specify the expected behavior of your requested feature. Some great and
helpful pointers are your expected interface (e.g. command patterns, simple
sketches of the user interface, etc).
```
`dialog_hestiaUI` is natively available for CSS-only application.

# Current Absent Behavior
```
Please specify the current behavior (e.g. workaround, blockage, etc).
```
`dialog_hestiaUI` is completely not available.

# Attachment
```
Please drag and drop the necessary data files (e.g. screenshot, logs, etc)
below.
```

**Comment by hollowaykeanho** [comment]
Implemented in 8f41c27f741d18ca579cc244031803d49c458953

**Comment by hollowaykeanho** [comment]
Released in https://github.com/ZORALab/Hestia/releases/tag/v1.0.0
